tests/fixtures/.gitignore

```diff
@@ -0,0 +1 @@
+/MutatedTestCase.php
```

Comment:
This ensures the fixtures directory exists, and that the weird mutated test class does not get commited.